packages/babylon-proto-ts/tsconfig.lib.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Ideally this should extend from the `tsconfig.base.json` at the root level but this library seems a bit tricky and it's ok to have its own tsconfig anyway.